packages/iov-ethereum/src/ethereumconnection.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { EthereumConnection } from "./ethereumconnection";
+import { TestConfig } from "./testconfig";
+
+const skipTests = (): boolean => !process.env.ETHEREUM_ENABLED;
+
+const pendingWithoutEthereum = () => {
```

Comment:
Could you use real functions instead of arrow function? I know this is how existing code is written, but we should use classic `function`s that follow a consistent style with classes and enums and are way easier to parse when reading a lot of code